### DISCUSSION TITLE
Feature request: The client `Test request` should display responses of content type `application/x-ndjson`

### DISCUSSION THREAD

**Discussion by achaphiv** [scalar/scalar#3885] [Ideas]
I have an endpoint giving back `Content-Type: application/x-ndjson`.

It's just multiple JSON objects separated by a new line.

E.G.

```json
{ "result": {} }
{ "result": {} }
{ "result": {} }
```

The UI currently displays `Binary file`.

![image](https://github.com/user-attachments/assets/15adb7b7-0b1a-4443-b943-1d2116666529)

I'd be okay with just displaying the content as raw text. But ideally each object would be pretty formatted.

**Comment by iamtimur** [comment]
I have the same problem with displaying the response, but in my case the server returns `application/vnd.api+json `

Is it possible to display response body preview with such Content-Type? 

![image](https://github.com/user-attachments/assets/d7eefa6b-f2a3-48c8-9442-dd2f29ad5e3a)

**Comment by hanspagel** [comment]
Finally, a PR

https://github.com/scalar/scalar/pull/9869
